test/school_house_web/controllers/lesson_controller_test.exs

```diff
@@ -29,5 +57,12 @@ defmodule SchoolHouseWeb.LessonControllerTest do
 
       assert body =~ "Page not found"
     end
+
+    test "redirects old routes to new routes", %{conn: conn} do
+      Enum.map(@routes_map, fn {{section, name}, {new_section, new_name}} ->
+        conn = get(conn, Routes.lesson_path(conn, :lesson, "en", section, name))
+        assert redirected_to(conn) =~ "/en/lessons/#{new_section}/#{new_name}"
+      end)
+    end
```

Comment:
🎉

Reply:
Thank you! 🙏

Reply:
@doomspork not urgent, but are you able to deploy prs that stem from branches outside the repo (like the branch used in this pr)? That way we can test out things like this in a deployed environment.

Reply:
Please lmk if there's a better way to initiate this PR to make it easier to test

Reply:
@BrooklinJazz you're good! My previous comment was more generally related to all prs from forks so anyone can check out the updates without needing to pull down the branch. Not a big deal, but since we started using [review apps](https://devcenter.heroku.com/articles/github-integration-review-apps) it would be nifty!